terraform/modules/ecs/ecs.tf

```diff
@@ -353,8 +369,42 @@ data "template_file" "logspout_container_definition" {
   }
 }
 
+/* task definition for the worker service */
+data "template_file" "worker_container_definition" {
+  template = "${file(var.ecs_service_worker_container_definition_file_path)}"
+
+  vars {
+    image           = "${aws_ecr_repository.default.repository_url}"
+    secret_key_base = "${var.secret_key_base}"
+    project_name    = "${var.project_name}"
+    task_name       = "${var.ecs_service_web_task_name}"
+    task_port       = "${var.ecs_service_worker_task_port}"
+
+    environment              = "${var.environment}"
+    rails_env                = "${var.rails_env}"
+    region                   = "${var.region}"
+    log_group                = "${var.aws_cloudwatch_log_group_name}"
+    database_user            = "${var.rds_username}"
+    database_password        = "${var.rds_password}"
+    database_url             = "${var.rds_address}"
+    elastic_search_url       = "${var.es_address}"
+    aws_elasticsearch_region = "${var.aws_elasticsearch_region}"
+    aws_elasticsearch_key    = "${var.aws_elasticsearch_key}"
+    aws_elasticsearch_secret = "${var.aws_elasticsearch_secret}"
+
+    rollbar_access_token = "${var.rollbar_access_token}"
+    redis_url            = "${var.redis_url}"
+
+    pp_transactions_by_channel_token = "${var.pp_transactions_by_channel_token}"
+    pp_user_satisfaction_token       = "${var.pp_user_satisfaction_token}"
+    domain                           = "${var.domain}"
+
```

Comment:
I think we probably want to add the `google_api_json_key` to the env vars here. What do you think?

Reply:
👍